core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -68,6 +68,8 @@ oppia.controller('CreatorDashboard', [
       smText: 'Publish the exploration to receive statistics.'
     };
 
+    var CurrentView = 'card';
```

Comment:
``` var activeView =  constants.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS.CARD); ``` 
Ditto below

Reply:
I have just initialized the variable because when I checked the value of viewType, the value came out to be card or list as selected by the user

Reply:
Yes, that true @suicide11 but in order to maintain the convention and uniformity throughout the codebase, the constants are tackled this way. 
You may want to consider a case where let's say ```card``` is changed to ```cards``` then you'll have to change all occurrences of ```card``` in the codebase. This practice is called DRY i.e. Don't Repeat Yourself.